### PR TITLE
New Repo: alibaba-cloud-controller-manager

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: core-services/prow/02_config/openshift/cloud-provider-alibaba-cloud/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+approve:
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/cloud-provider-alibaba-cloud/_pluginconfig.yaml:1]: I thought this used to all go in one big file? Has this changed?

Comment (Fedosin) [reply core-services/prow/02_config/openshift/cloud-provider-alibaba-cloud/_pluginconfig.yaml:1]: it was generated by `make new-repo`.

**Pull request merged**